cylc/uiserver/resolvers.py

```diff
@@ -235,17 +235,20 @@ async def clean(cls, workflows, args, workflows_mgr, executor, log):
             clean_func = partial(_clean, tokens, opts)
             try:
                 log.info(f'Cleaning {tokens}')
-                future = await asyncio.wrap_future(executor.submit(clean_func))
+                string = await asyncio.wrap_future(
+                    executor.submit(clean_func)
+                )
             except Exception as exc:
                 log.exception(exc)
                 return cls._error(exc)
             else:
-                if future == WORKFLOW_RUNNING_MSG:
+                if string == WORKFLOW_RUNNING_MSG:
                     log.error(ServiceFileError(WORKFLOW_RUNNING_MSG))
                     return cls._error(WORKFLOW_RUNNING_MSG)
+                log.info(string)
 
         # trigger a re-scan
-        await workflows_mgr.update()
+        await workflows_mgr.scan()
```

Comment:
This is due to a change in the scanning logic, the `.scan` interface requests rather than performs a scan preventing concurrent scan operations from fighting over the same workflows.